Thing.h

```diff
@@ -360,6 +360,10 @@ class ThingProperty : public ThingItem {
       callback(newValue);
     }
   }
+  bool isInRange(ThingDataValue value){
+    return (this->maximum < this->minimum) //to check if minimum and maximum values are asigned
+    || (this->maximum > value.)
```

Comment:
why dot in value. ?

Reply:
where is isInRange used ?

Reply:
Sorry, at first I was going to handle the process of validation the range in a function then decide to do it directly but forget to delete the function. this function is not used and should delete.

Reply:
should I just create another pull request?